views/base.njk

```diff
@@ -68,7 +68,13 @@
   }) }}
 {% endblock %}
 
-{% block content %}{% endblock %}
+{% block content %}
+  <div class="govuk-width-container">
```

Comment:
Question: What is your opinion on not wrapping the `pageContent` block in these classes `<div class="govuk-width-container"> <div class="govuk-body">`

If we define classes `<div class="govuk-width-container"> <div class="govuk-body">` within `pageContent` it gives us more flexibility, for when we ever need to wrap `pageContent` in something different?